applications/stream-applications-core/pom.xml

```diff
@@ -19,13 +19,13 @@
         <revision>3.2.1-SNAPSHOT</revision>
         <stream-apps-core.version>${revision}</stream-apps-core.version>
         <java-functions.version>1.2.0</java-functions.version>
-        <apps.base-image>springcloud/baseimage:1.0.0</apps.base-image>
+        <apps.base-image>springcloud/baseimage:1.0.1</apps.base-image>
         <mockserver.version>5.10</mockserver.version>
         <spring-cloud-stream-dependencies.version>3.2.2</spring-cloud-stream-dependencies.version>
         <spring-cloud-stream.version>3.2.2</spring-cloud-stream.version>
-        <spring-cloud-dataflow-apps-generator-plugin.version>1.0.6</spring-cloud-dataflow-apps-generator-plugin.version>
-        <spring-cloud-dataflow-apps-docs-plugin.version>1.0.6</spring-cloud-dataflow-apps-docs-plugin.version>
-        <spring-cloud-dataflow-apps-metadata-plugin.version>1.0.6</spring-cloud-dataflow-apps-metadata-plugin.version>
+        <spring-cloud-dataflow-apps-generator-plugin.version>1.0.7-SNAPSHOT</spring-cloud-dataflow-apps-generator-plugin.version>
```

Comment:
This change is unrelated, correct?

Reply:
This change is needed to ensure that the new version of app plugin is used so that baseimage version can be specified in stream-applications-core.